software/spmd/bsg_cuda_lite_runtime/dma/Makefile

```diff
@@ -0,0 +1,34 @@
+#########################################################
+# Network Configutaion
+# If not configured, Will use default Values
+	bsg_global_X ?= $(bsg_tiles_X)
+	bsg_global_Y ?= $(bsg_tiles_Y)+1
```

Comment:
Did this sneak in?

Reply:
The file? This is a simple test that I added.

Reply:
One PR one feature...

Reply:
I vote that we be agile in this circumstance.